std/format.d

```diff
@@ -64,24 +64,31 @@ import std.range.primitives;
 import std.traits;
 
 
-/**********************************************************************
- * Signals a mismatch between a format and its corresponding argument.
+/**
+Signals a mismatch between a format and its corresponding argument.
  */
 class FormatException : Exception
 {
-    @safe pure nothrow
+    @safe @nogc pure nothrow
     this()
     {
         super("format error");
     }
 
-    @safe pure nothrow
+    @safe @nogc pure nothrow
```

Comment:
Interesting that the compiler accepts this (though it doesn't make so much sense, does it?)

Reply:
Exception doesn't anything. `new` does that. You can easily use stdx.allocator.make and free to throw and catch this exception.